public/javascripts/notebook/job_progress.js

```diff
@@ -65,7 +83,12 @@ define([
         };
 
         var runningJobs = _.map(_.reject(jobsProgress, isCompleted));
-        var runningJobsInfo = _.flatten(_.map(runningJobs, function(p){
+        var runningJobsInfo = _.flatten(_.map(runningJobs, function(p) {
+          console.log("job group: " + p.group);
+          if (p.group) {
+            highlightCell(cells, p.group.substring("cell-".length));
```

Comment:
this `cell-$id` format should be abstracted, so maybe provide a `p.cell_id` instead of `group`.